packages/wdio-browserstack-service/src/launcher.ts

```diff
@@ -150,7 +157,7 @@ export default class BrowserstackLauncherService implements Services.ServiceInst
         this._options.accessibility = this._accessibilityAutomation
 
         // by default observability will be true unless specified as false
-        this._options.testObservability = this._options.testObservability === false ? false : true
+        this._options.testObservability = this._options.testObservability !== false
```

Comment:
`undefined !== false` this returns true
Should we use `!!this._options.testObservability` to have a boolean value

Reply:
This is mistakenly changed, not supposed to be changed.

Can't use `!!this._options.testObservability` as null or undefined will evaluate to false, have changed it to `this._options.testObservability !== false`